governance/README.md

```diff
@@ -79,3 +79,21 @@ A **TSC Member** nominates a person in the private *TSC* channel in Zulip explai
 
 People will be removed from this team after one year of no activity, and would be recognized as **Emeritus TSC Members**. A majority of **TSC Members** can also vote to remove a **TSC Member**.
 
+## Administrators
+
+This is a smaller group of people group of people (maximum 5) that have administrator permissions in the Servo organization and repositories. Administrators would be also the [organization owners](https://github.com/orgs/servo/people?query=role%3Aowner).
+
+[Linux Foundation Europe](https://github.com/LF-Europe) would be always an extra administrator and owner while the project is hosted there.
+
+### Requirements
+
+To be recognized with **Administrator** status, a community member must demonstrate to be trusted as a **TSC Member** of the Servo project for a significant amount of time.
```

Comment:
So administrator is not necessarily TSC member, it is just trusted as a TSC member?

Reply:
Mmm, maybe the wording is wrong, I think the original idea was that administrators have to be TSC members. They have to be a TSC member for a significant amount of time which is trusted by the Servo community.

What if we reword it to:
```suggestion
To be recognized with **Administrator** status, a community member must be a **TSC Member** of the Servo project for a significant amount of time who has demonstrated to be trusted by the Servo community.
```

Reply:
Maybe instead of "community member" we should just say "TSC Member"?

Disclaimer: I am also not native speaker.